test/powershell/Modules/Microsoft.PowerShell.Management/Alias.Tests.ps1

```diff
@@ -1,55 +1,112 @@
-    }
-    
-    It "Test number of alias not Zero" {
-        $aliases = Get-ChildItem Alias:\
-        $aliases.Count | Should Not Be 0
-    }
+        AfterEach {
+            Remove-Item -Path "Alias:\${testAliasName}" -Force -ErrorAction SilentlyContinue
+        }
 
-    It "Test alias 'dir'" {
-        $dirAlias = Get-Item Alias:\dir
-        $dirAlias.CommandType | Should Be 'Alias'
-        $dirAlias.Name | Should Be 'dir'
-        $dirAlias.Definition | Should Be 'Get-ChildItem'
-    }
+        It "Test number of alias not Zero" {
+            $aliases = @(Get-ChildItem "Alias:\")
+            $aliases.Count | Should Not Be 0
+        }
 
-    It "Test creating new alias" {
-        $newAlias = New-Item -Path Alias:\ -Name 'NewTestAlias' -Value $testAliasValue
-        try {
-            $newAlias.CommandType | Should Be 'Alias'
-            $newAlias.Name | Should Be 'NewTestAlias'
-            $newAlias.Definition | Should Be $testAliasValue
+        It "Test alias dir" {
+            $dirAlias = Get-Item "Alias:\dir"
+            $dirAlias.CommandType | Should Be "Alias"
+            $dirAlias.Name | Should Be "dir"
+            $dirAlias.Definition | Should Be "Get-ChildItem"
         }
-        finally {
-            Remove-Item -Path 'Alias:\NewTestAlias' -ErrorAction SilentlyContinue
+
+        It "Test creating new alias" {
+            try {
+                $newAlias = New-Item -Path "Alias:\" -Name "NewTestAlias" -Value $testAliasValue
+                $newAlias.CommandType | Should Be "Alias"
+                $newAlias.Name | Should Be "NewTestAlias"
+                $newAlias.Definition | Should Be $testAliasValue
+            }
+            finally {
+                Remove-Item -Path "Alias:\NewTestAlias" -Force -ErrorAction SilentlyContinue
+            }
         }
-    }
 
-    It "Test get-item on alias provider" {
-        $alias = Get-Item -Path "Alias:\${testAliasName}"
-        $alias.CommandType | Should Be 'Alias'
-        $alias.Name | Should Be $testAliasName
-        $alias.Definition | Should Be $testAliasValue
-    }
+        It "Test Get-Item on alias provider" {
+            $alias = Get-Item -Path "Alias:\${testAliasName}"
+            $alias.CommandType | Should Be "Alias"
+            $alias.Name | Should Be $testAliasName
+            $alias.Definition | Should Be $testAliasValue
+        }
+
+        It "Test Test-Path on alias provider" {
+            $aliasExists = Test-Path "Alias:\testAlias"
+            $aliasExists | Should Be $true
+        }
 
-    It "Test test-path on alias provider" {
-        $aliasExists = Test-Path Alias:\testAlias
-        $aliasExists | Should Be $true
+        It "Test executing the new alias" {
+            $result = Invoke-Expression $testAliasName
+            $result.GetType().Name | Should Be "DateTime"
+        }
     }
+}
+
+Describe "Extended Alias Provider Tests" -Tags "Feature" {
+    Context "Valdiation of Set-Item parameters for the Alias Provider" {
```

Comment:
I plan to add more cmdlet coverage to the "Extended Alias Provider Tests" describe block in the future. I was preparing for that future by putting the context blocks in now.
If you still think that I should remove the Context block please let me know.